Controllers/SystemUserController.cs

```diff
@@ -98,7 +98,6 @@ public async Task<IActionResult> Edit(string id, [Bind("Id,Name,StartDate,Descri
                 try
                 {
                     _context.Update(user);
-                    await _context.SaveChangesAsync();
```

Comment:
If we don't do this here, a DbUpdateConcurrencyException will break the entire page regardless and the try catch becomes redundant.

Reply:
It breaks when it is in the Try method, but it works when it is outside.

Reply:
Okay, that's odd. I suppose this will have to do anyway,